### PR TITLE
Support for classes from rms package

### PR DESCRIPTION
**Pull request by RomanTsegelskyi**
This pull request is aimed to add support for classes from rms #207. More classes will be added with time

### CODE REVIEW

**Comment by daroczig** [comment]
@RomanTsegelskyi, didn't we change for codecov?

File: R/S3.R [hunk 1]
```diff
@@ -1720,3 +1720,91 @@ pander.nls <- function(x, digits = panderOptions('digits'), show.convergence = F
...
+#' @param x an ols object
+#' @param long if to print the correlation matrix of parameter estimates. default(\code{FALSE})
+#' @param coefs if to the table of model coefficients, standard errors, etc. default(\code{TRUE})
+#' @param digits passed to \code{format}. Can be a vector specifying values for each column (has to be the same length as number of columns).
+#' @param round passed to \code{round}. Can be a vector specifying values for each column (has to be the same length as number of columns). Values for non-numeric columns will be disregarded.
+#' @param ... optional parameters passed to raw \code{pandoc.table} function
+#' @export
+pander.ols <- function (x, long = FALSE, coefs = TRUE,
+                        digits = panderOptions("digits"), round = panderOptions("round"), ...) {
+    requireNamespace("rms", quietly = TRUE)
```
Comment (daroczig) [inline R/S3.R]: We should also add this to the list of suggested packages, right?

Comment (RomanTsegelskyi) [inline R/S3.R]: Sure. This was just to agree on preferred output and then I would add tests and DESCRIPTION

**Comment by RomanTsegelskyi** [comment]
Yes, but I haven't merged the changes with switch to codecov yet. Should I merge and rebase this PR?

**Comment by daroczig** [comment]
Ah, OK. No need to rebase, this is fine.

**Pull request closed without merging**